bitnami/wordpress/values.yaml

```diff
@@ -370,6 +376,14 @@ livenessProbe:
   timeoutSeconds: 5
   failureThreshold: 6
   successThreshold: 1
+## @param readinessProbe.enabled Enable readinessProbe
+## @skip readinessProbe.httpGet[object]
```

Comment:
If the parameter is skipped it does not need the `[object]` modifier since it will not appear in the table